frontend/src/views/OverviewView.vue

```diff
@@ -0,0 +1,419 @@
+}
+function makeFilterVisible(
+  name:
+    | "client"
+    | "article"
+    | "orderStatus"
+    | "invoiceDate"
+    | "offerDate"
+    | "overdueDate"
+    | "serviceDate"
+    | "cashDiscount"
+    | "invoicePaymentTerm"
+    | "invoicePaymentStatus"
+    | "offerValidity"
+    | "offerStatus"
+    | "overdueNoticeLevel"
+    | "overduePaymentTerm"
+    | "overdueStatus",
+) {
+  filters.value[name].active = true;
+}
+function findClient(item: Client | undefined) {
+  filters.value.client.value = item;
+}
+
+function findArticle(item: Article | undefined) {
+  filters.value.article.value = item;
+}
+
+function findOrderStatus(item: OrderStatus | undefined) {
+  filters.value.orderStatus.value = item;
+}
+function findDate(dateFrom: Date | undefined, dateAfter: Date | undefined, dateId: string) {
+  switch (dateId) {
+    case "invoiceDate":
+      filters.value.invoiceDate.value = [dateFrom, dateAfter];
+      break;
+    case "offerDate":
+      filters.value.offerDate.value = [dateFrom, dateAfter];
+
+      break;
+    case "overdueDate":
+      filters.value.overdueDate.value = [dateFrom, dateAfter];
+      break;
+    case "serviceDate":
+      filters.value.serviceDate.value = [dateFrom, dateAfter];
+      break;
+    case "offerValidity":
+      filters.value.offerValidity.value = [dateFrom, dateAfter];
+
+      break;
+    case "overduePaymentTerm":
+      filters.value.overduePaymentTerm.value = [dateFrom, dateAfter];
+      break;
+  }
+}
+function findCashDiscount(item: String | undefined) {
+  filters.value.cashDiscount.value = item;
+}
+function findInvoicePaymentStatus(item: PaymentStatus | undefined) {
+  filters.value.invoicePaymentStatus.value = item;
+}
+function findOfferStatus(item: OfferStatus | undefined) {
+  filters.value.offerStatus.value = item;
+}
+function findOverdueNoticeLevel(item: OverdueNoticeLevel | undefined) {
+  filters.value.overdueNoticeLevel.value = item;
+}
+function findOverdueStatus(item: OverdueNoticePaymentStatus | undefined) {
+  filters.value.overdueStatus.value = item;
+}
+watch(
+  filters,
+  () => {
+    console.log(filters.value);
+  },
+  { deep: true },
+);
+onMounted(async () => {
+  reloadPage();
+});
+</script>
+
+<template>
+  <!-- TODO: assess if this page should be hidden for some user roles -->
+  <div class="flex flex-col overflow-hidden">
+    <div class="card flex w-full gap-x-6 mb-3">
+      <span class="relative grow">
+        <i
+          class="pi pi-search absolute top-2/4 -mt-2 left-3 text-surface-400 dark:text-surface-600"
+        />
+        <InputText v-model="search" placeholder="Suche" class="pl-10 w-full" />
+      </span>
+
+      <Button icon="pi pi-times" size="small" severity="danger" rounded />
+    </div>
+    <div class="font-bold">Filterregeln:</div>
+  </div>
+  <SplitButton label="Options" :model="items" :class="'w-full'" />
+  <section class="flex flex-col sm:flex-row gap-4 sm:gap-4 Class Properties flex-wrap">
```

Comment:
Looks like there is some overlap

<img width="270" alt="image" src="https://github.com/user-attachments/assets/d15b3fc7-7c4f-4a72-bc76-4f920995ecbb">